src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/_help.py

```diff
@@ -37,3 +37,9 @@
     type: group
     short-summary: Manage Redis patch schedules.
 """
+
+helps['redis patch-schedule patch-schedule'] = """
+    type: group
+    short-summary: This group will soon be deprecated. The command, `redis patch-schedule patch-schedule show`,
```

Comment:
May be worthwhile indicating the CLI version that it will be removed in and creating a GH issue with a milestone of that release so we remember to actually remove it later.